tests/unit/completion/test_completionwidget.py

```diff
@@ -218,3 +229,29 @@ def test_completion_show(show, rows, quick_complete, completionview,
     completionview.set_model(None)
     completionview.completion_item_focus('next')
     assert not completionview.isVisible()
+
+
+def test_completion_item_del(completionview, simplemodel):
+    """Test that completion_item_del invokes delete_cur_item in the model."""
+    simplemodel.srcmodel.delete_cur_item = unittest.mock.Mock()
```

Comment:
Why not simply do `model = base.CompletionModel(delete_cur_item=unittest.mock.Mock())` or so in the `simplemodel` fixture?